firmware/integration/rusefi_config.txt

```diff
@@ -1397,7 +1398,12 @@ float[CLT_CURVE_SIZE] iacCoasting;    CLT-based idle position for coasting (used
 	linear_sensor_s highPressureFuel;
 	linear_sensor_s lowPressureFuel;
 
-uint8_t[536] unused3328;;"units", 1, 0, -20, 100, 0
+! issue 455: RPM hard limit depending on CLT
+int8_t[CLT_LIMITER_CURVE_SIZE] cltRevLimitRpmBins;CLT-based target RPM for hard limit depending on CLT LFAstyle;"C",        1,     0,   -70,    120,  0
+uint16_t[CLT_LIMITER_CURVE_SIZE] cltRevLimitRpm;See idleRpmPid;"",      1,     0,      0,    8000,   0
+
+! was [536] - CLT_LIMITER_CURVE_SIZE*size_of(int8_t) [4*1] - CLT_LIMITER_CURVE_SIZE*size_of(uint16_t) [4*2] = 536 - 4 - 8 = 524
```

Comment:
we don't need to record what the old line was here

Reply:
Ok, will clear this comments